.github/workflows/relay.yml

```diff
@@ -106,6 +110,7 @@ jobs:
         # https://github.com/moby/buildkit/issues/1896
         # Without the change some jobs were failing with `no space left on device`
         name: Move cache
+        if: github.event_name != 'schedule'
```

Comment:
If we would build an image for `schedule event, this line would need to be removed.

Reply:
5f76e08